apps/test/unit/studio/itemTest.js

```diff
@@ -0,0 +1,92 @@
+import {expect} from '../../util/configuredChai';
+import Studio from '@cdo/apps/studio/studio';
+
+import {
+  Direction,
+  BEHAVIOR_CHASE,
+  BEHAVIOR_FLEE,
+  BEHAVIOR_STOP,
+  BEHAVIOR_WANDER,
+  BEHAVIOR_WATCH_ACTOR,
+  BEHAVIOR_GRID_ALIGNED
+} from '@cdo/apps/studio/constants';
+
+import Item from '@cdo/apps/studio/Item';
+
+describe('item', () => {
+  before(() => {
+    Studio.trackedBehavior = {};
+    Studio.trackedBehavior.createdItems = {};
+  });
+
+  let item;
+
+  describe('update', () => {
+    describe('if destination is set', () => {
```

Comment:
sort of unnecessary, but it is the case that `update` can have some pretty different behavior based on when in the "lifecycle" of destination determination it's called, and I wanted to make absolutely clear which case we're in